### ISSUE TITLE
A few things

### ISSUE THREAD

**Issue by b-t-k** [PiperKev/richard-henry-dana-jr_to-cuba-and-back#1]
*note* I am not sure its necessary to make each hyphenation change its own separate editorial commit. It would be easier to look at them all one chapter at a time.

- run clean
- run typogrify
- run modernize-spelling
- run semanticate
- run build-toc

- Remove italics: `<i xml:lang="es">matador</i>`
-  am not sure about the need for italics here. But I will leave it up to you as you understand the context more. `<i xml:lang="es">mayoral</i>` vs mayordomo?

## epigraph.xhtml
- remove all caps

## chapter 11
- Remove the ligature in `<i xml:lang="la">lusus naturæ</i> `
- add abbr to Troy, N. York.

## colophon
- remove double period add abbr: `Richard Henry Dana, Jr.</a>.`

**Comment by PiperKev** [comment]
Corrected issues with a couple of exceptions.

Italics on "matador."  I was instructed by Alex Cabal to drop the parenthesis and italicize it:  
"Remove parens, add italics. Even though this is English now, it literally means "killer" so the juxtaposition with its definition would make italics appropriate."

Typogrify changed the two-em-dashes for partially obscured names (mostly after Senor, Father and Miss) to hyphens, so I had to correct them back to two-em-dashes before committing.  I also caught that I used two-em-dashes for fully obscured names, so I corrected those to three-em-dashes.

I italicized "mayoral" because that Spanish word for "farm manager" isn't in M-W, and neither is "contra-mayorales." "Mayordomo" IS in M-W, so I didn't italicize it.
